ar_exercise/exercise.rb

```diff
@@ -32,17 +32,40 @@ class Apartment < ActiveRecord::Base
 # Use Active record to do the following, and store the results **in a variable**
 # example: get every tenant in the DB
 all_tenants = Tenant.all
+#
+# # get the first tenant in the DB
+first_tenant = Tenant.first
+        #
+# # get all tenants older than 65
+tenant_older_65 = Tenant.where("age >= ?", 65)
+#
+# # get all apartments whose price is greater than $2300
+expensive_apt = Apartment.where("monthly_rent >= ?", 2300)
+#
+# # get the apartment with the address "6005 Damien Corners"
+damien_corners = Apartment.where(address: "6005 Damien Corners")
+damien_corners = Apartment.find_by(:address => "6005 Damien Corners")
+#
 
-# get the first tenant in the DB
-# get all tenants older than 65
-# get all apartments whose price is greater than $2300
-# get the apartment with the address "6005 Damien Corners"
 # get all tenants in that apartment
+damien_corners = Apartment.find_by(:address => "6005 Damien Corners")
+all_tenants_Damien_Corners = Apartment.where(""apartment_id == tenant)
 
 # Use `each` and `puts` to:
 # Display the name and ID # of every tenant
+Tenant.find_each do |tenant|
+  puts "Tenant name: #{tenant.name}, Tenant id: #{tenant.apartment_id}"  end
+
```

Comment:
i think this will work, better might be: 

```ruby
Tenant.all.each do |tenant|

end
```

I think `find_each` is more valuable if you're querying for specific things in our database